tests/simulation/params/test_validators_params.py

```diff
@@ -97,3 +105,92 @@ def test_numerical_dissipation_mode_validator(
+        initial_condition=HeatEquationInitialCondition(temperature="10"),
+    )
+    solid_model_without_initial_condition = Solid(
+        volumes=[GenericVolume(name="CHTSolid")],
+        material=aluminum,
+        volumetric_heat_source="0",
+    )
+    solid_model_without_specific_heat_capacity = Solid(
+        volumes=[GenericVolume(name="CHTSolid")],
+        material=SolidMaterial(
+            name="aluminum_without_specific_heat_capacity",
+            thermal_conductivity=235 * u.kg / u.s**3 * u.m / u.K,
+            density=2710 * u.kg / u.m**3,
+        ),
+        volumetric_heat_source="0",
+        initial_condition=HeatEquationInitialCondition(temperature="10;"),
+    )
+    surface_output_with_residual_heat_solver = SurfaceOutput(
+        name="surface", write_single_file=True, output_fields=["residualHeatSolver"]
+    )
+
+    # Valid simulation params
+    with SI_unit_system:
+        params = SimulationParams(
+            models=[fluid_model, solid_model],
+            time_stepping=timestepping_unsteady,
+            outputs=[surface_output_with_residual_heat_solver],
+        )
+
+    assert params
+
+    message = (
+        "Heat equation output variables: residualHeatSolver is requested in "
+        f"{surface_output_with_residual_heat_solver.output_type} with no definition of solid models."
+    )
+
+    # Invalid simulation params
+    with SI_unit_system, pytest.raises(ValueError, match=re.escape(message)):
+        _ = SimulationParams(
+            models=[fluid_model],
+            time_stepping=timestepping_unsteady,
+            outputs=[surface_output_with_residual_heat_solver],
+        )
+
+    message = "In solid model, the heat capacity needs to be specified for unsteady simulations."
+
+    # Invalid simulation params
+    with SI_unit_system, pytest.raises(ValueError, match=re.escape(message)):
+        _ = SimulationParams(
+            models=[fluid_model, solid_model_without_specific_heat_capacity],
+            time_stepping=timestepping_unsteady,
+            outputs=[surface_output_with_residual_heat_solver],
+        )
+
+    message = (
+        "In solid model, the initial condition needs to be specified for unsteady simulations."
+    )
+
+    # Invalid simulation params
+    with SI_unit_system, pytest.raises(ValueError, match=re.escape(message)):
+        _ = SimulationParams(
+            models=[fluid_model, solid_model_without_initial_condition],
+            time_stepping=timestepping_unsteady,
+            outputs=[surface_output_with_residual_heat_solver],
+        )
+
+    message = "In solid model, the initial condition needs to be specified for initialization with expressions."
+
+    # Invalid simulation params
+    with SI_unit_system, pytest.raises(ValueError, match=re.escape(message)):
+        _ = SimulationParams(
+            models=[fluid_model_with_initial_condition, solid_model_without_initial_condition],
+            time_stepping=timestepping_steady,
```

Comment:
Just inline the code here.
```suggestion
            time_stepping=Steady(),
```

Reply:
Updated.